src/lib.rs

```diff
@@ -334,6 +335,7 @@ where
     N: Zero,
 {
     /// Create a new, empty `Counter`
+    #[must_use]
```

Comment:
This does not seem related to splitting the tests?

Reply:
ah, i think some of my `cargo clippy` changes seaped through.
should i remove it?

Reply:
this is useful for lint and static analysis purposes, but if you don't want this **here** or **at all** I'll remove it regardless

Reply:
I'm not entirely opposed to the annotation, but I have a strong preference for focused PRs which do one thing. Makes it easier to identify the root causes for changes after the fact, when looking at the conversation attached to the work. This should be in a separate (possibly tiny) PR with other lint-related changes, if there are any.

Reply:
understood